application_sdk/inputs/deployment_config.py

```diff
@@ -0,0 +1,37 @@
+"""Minimal deployment configuration manager."""
+
+from typing import Any, Dict, Optional
+
+from application_sdk.constants import (
+    DEPLOYMENT_SECRET_PATH,
+    DEPLOYMENT_SECRET_STORE_NAME,
+)
+from application_sdk.inputs.secretstore import SecretStoreInput
+from application_sdk.observability.logger_adaptor import get_logger
+
+logger = get_logger(__name__)
+
+
+class DeploymentConfig:
+    """Simple deployment config manager with caching."""
+
+    _config: Optional[Dict[str, Any]] = None
+
+    @classmethod
+    async def get(cls, force_refresh: bool = False) -> Dict[str, Any]:
```

Comment:
as this is just another call to SecretStoreInput - why can't this be a method there itself? `SecretStoreInput.get_deployment_secret()`